manuskript/load_save/version_1.py

```diff
@@ -663,6 +655,9 @@ def loadProject(project, zip=None):
                 if f[-4:] in [".xml", "opml"]:
                     with open(os.path.join(dirpath, f), "rb") as fo:
                         files[os.path.join(p, f)] = fo.read()
+                elif f[-4:] in [".png", "jpeg",".jpg","bmp","tiff",".gif",".pdf"]:
```

Comment:
This should by configurable, I use often SVG images.

Also HTML Files often with research information or other filetypes. (bst -> bibtex and so on)


In the moment I store my Images in folder on the same level as the project-file (I do not us the zip/one-file mode!)... So I don't have this Problem.